framework/bootstrap/http.go

```diff
@@ -17,21 +17,60 @@ import (
 type PathHandler struct {
 	mux     *http.ServeMux
 	mapping map[string]http.Handler
+	// realPath -> redirectPath mapping
+	pathRedirects map[string]string
 	sync.RWMutex
 }
 
-func NewPathHandler() *PathHandler {
+func NewPathHandler(pathRedirects map[string]string) *PathHandler {
+	// from redirectPath -> realPath to realPath -> redirectPath
+	var reversePathRedirects map[string]string
```

Comment:
reversePathRedirects maybe "map[string][]string" to support "A->C,B->C" ?

Reply:
fixed